applications/ecallmgr/src/ecallmgr_call_command.erl

```diff
@@ -1529,3 +1536,48 @@ tone_duration_on(Tone) ->
 -spec tone_duration_off(kz_json:object()) -> ne_binary().
 tone_duration_off(Tone) ->
     kz_json:get_binary_value(<<"Duration-OFF">>, Tone).
+
+-spec transfer(atom(), ne_binary(), kz_json:object()) -> {ne_binary(), ne_binary()}.
+transfer(Node, UUID, JObj) ->
+    TransferType = kz_json:get_value(<<"Transfer-Type">>, JObj),
+    TransferTo = kz_json:get_value(<<"Transfer-To">>, JObj),
+    transfer(Node, UUID, TransferType, TransferTo, JObj).
+
+-spec transfer(atom(), ne_binary(), ne_binary(), ne_binary(), kz_json:object()) -> {ne_binary(), ne_binary()}.
+transfer(Node, UUID, <<"attended">>, TransferTo, JObj) ->
+    CCVs = kz_json:to_proplist(kz_json:get_value(<<"Custom-Channel-Vars">>, JObj, kz_json:new())),
+    CCVList = [<<"Account-ID">>
+              ,<<"Authorizing-ID">>
+              ,<<"Authorizing-Type">>
+              ,<<"Channel-Authorized">>
+              ],
+    Realm = props:get_value(<<"Account-Realm">>, CCVs),
+    ReqURI = <<TransferTo/binary, "@", Realm/binary>>,
+    Vars = [{<<"Ignore-Early-Media">>, <<"ring_ready">>}
+           ,{<<"Simplify-Loopback">>, <<"false">>}
+           ,{<<"Loopback-Bowout">>, <<"true">>}
+           ,{<<"Loopback-Request-URI">>, ReqURI}
+           ,{<<"SIP-Invite-Domain">>, Realm}
+           ],
+    Props = [KV || {K,_V} = KV <- CCVs,
+                   lists:member(K, CCVList)
+            ]  ++ Vars,
+    [Export | Exports] = ecallmgr_util:process_fs_kv(Node, UUID, Props, 'set'),
+    Arg = list_to_binary([Export
+                         ,[[",", Exported] || Exported <- Exports]
+                         ]
+                        ),
+    {<<"att_xfer">>, list_to_binary(["{", Arg, "}loopback/", TransferTo, <<"/">>, transfer_context(JObj)])};
```

Comment:
Looks like Arg does not need to be turned into binary as it will be done here. Or maybe with iolist_to_binary